ItemLibrary/Smartphone.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ItemLibrary
+{
+    public class Smartphone : Item
+    {
+        public string FrontCamera { get; set; }
+        public string EndCamera { get; set; }
+        public double ScreenDiagonal { get; set; }
```

Comment:
could be display size as it is in computer, but I don't think that is a problem

Reply:
added DispalySize field